doc/guides/source/UserGuide/Protection.rst

```diff
@@ -128,17 +128,14 @@ By default, all objects in Indico are INHERITING.
 Domain Control Policy
 ---------------------
 
-If an Indico object (category, event, session, contribution,
-material, file and link) is PUBLIC, we can restrict the access to
-users accessing Indico from some given IPs (these IPs could be like
-127.1 which means that every IP starting like this will be valid).
-
-If the Indico object is RESTRICTED, this checking will not be
-applied.
-
-If it is INHERITING, it will have the same access protection as its
-parent. Its access protection status will therefore change whenever
-the parent's access protection changes.
+Access to an Indico object (category, event, session, contribution,
+material, file and link) can be restricted to a range of IP addresses.
+To do that, the *Protection mode* of the Indico object needs to be set to
+**Protected**. Then, the desired IP network can be selected from the
```

Comment:
Alright. BTW, currently we allow networks only in categories and events. So the statement `Access to an Indico object (category, event, session, contribution,
+material, file and link) can be restricted to a range of IP addresses` is technically wrong. I'll update it to only cover categories and events.